html/canvas/element/text-styles/META.yml

```diff
@@ -22,3 +22,13 @@ links:
           subtest: System fonts must be computed to explicit values
         - test: 2d.text.font.parse.complex.html
           subtest: 'Canvas test: 2d.text.font.parse.complex'
+    - product: chrome
+      url: https://crbug.com/1066953
+      results:
+        - test: 2d.text.draw.baseline.hanging.html
+        - test: 2d.text.draw.baseline.ideographic.html
+        - test: 2d.text.draw.space.collapse.end.html
+        - test: 2d.text.draw.space.collapse.other.html
+        - test: 2d.text.draw.space.collapse.space.html
+        - test: 2d.text.draw.space.collapse.start.html
+        - test: 2d.text.measure.width.space.html
```

Comment:
The bug is about this one. The other tests were updated, but it sounds like it was a problem with the tests, fonts not loading. Have you looked at the failure mode of the other tests to see that they also seem to be about whitespace collapsing?

Reply:
Update: I'll check a few of them.